backend/wmg/data/schemas/cube_schema.py

```diff
@@ -11,6 +11,12 @@
     "organism_ontology_term_id",
 ]
 
+expression_summary_fmg_indexed_dims = [
```

Comment:
The [spec](https://docs.google.com/document/d/1gDx3Ad3b0uvvsXzaRFsKAeBAUkzH_EPrqwZv66c7k3E/edit#heading=h.nn44ej8p8r1b) says this is the order `[organism_ontology_term_id, tissue_ontology_term_id, cell_type_ontology_term_id]`

Reply:
good catch, spec was outdated.